### ISSUE TITLE
DEPRECATION WARNING: The `check_pending!` method is deprecated in favor of `check_all_pending!`

### ISSUE THREAD

**Issue by trombini77** [ianheggie/health_check#143]
I got this warning using:

Ruby 3.3.0 (container image ruby:3.3.0-slim)
Rails 7.1.3
health_check 3.1.0

WARNING LOG:
```
 WARN -- : DEPRECATION WARNING: The `check_pending!` method is deprecated in favor of `check_all_pending!`. The new implementation will loop through all available database configurations and find pending migrations. The prior implementation did not permit this. (called from block in process_checks at /app/vendor/bundle/ruby/3.3.0/gems/health_check-3.1.0/lib/health_check/utils.rb:37)
```

App Path call: `http://localhost/health_check/full.json`

`config/initializers/health_check.rb` file line:

```
config.full_checks = ["site", "database", "migrations", "redis", "sidekiq-redis", "rabbitmq"]
```

**Comment by muhammadnawzad** [comment]
Seems to be solved in https://github.com/Purple-Devs/health_check/pull/140. However, it is still a pending pull request.
